src/Troopers/BehatContexts/Context/ExtendedEntityContext.php

```diff
@@ -39,4 +40,57 @@ public function beforeScenario($event)
             }
         }
     }
+
+    /**
+     * @Then /^ I should find at least one (.*) like:?$/
+     *
+     * @param $nbr
+     * @param $name
+     * @param TableNode $table
+     *
+     * @throws \Exception
+     */
+    public function existAtLeastOneLikeFollowing($name, TableNode $table)
+    {
+        $entityName = $this->resolveEntity($name)->getName();
+
+        $rows = $table->getRows();
+        $headers = array_shift($rows);
+
+        $row = array_shift($rows);
+        $object = $this->getEntityManager()
+            ->getRepository($entityName)
+            ->findOneBy(
+                $this->getQueryParams($entityName, $headers, $row)
+            );
+
+        if (is_null($object)) {
+            throw new \Exception(sprintf('There is not any object for the following params: %s', json_encode($this->getQueryParams($entityName, $headers, $row))));
```

Comment:
What is it supposed to echo ? Something like this I guess :
```
There is not any object for the following params: {"firstname":"Abitlabol","enabled":false}
```

I guess we could be more pretty 💋  but it's ok for developers